unsplasharp/Source/UnsplasharpClient.cs

```diff
@@ -914,7 +891,7 @@ public enum Resolution {
                 Photos = double.Parse((string)data["photos"]),
                 Downloads = double.Parse((string)data["downloads"]),
                 Views = double.Parse((string)data["views"]),
-                Likes = double.Parse((string)data["likes"]),
+                Likes = double.Parse((string)data["likes"] ?? "0"),
```

Comment:
The payload is missing likes in some cases, so let's change it to 0 in that case.